source/base/function_lib.cc

```diff
@@ -2729,11 +2728,11 @@ namespace Functions
         const double delta_x =
           ((interval_endpoints[d].second - interval_endpoints[d].first) /
            n_subintervals[d]);
-        p_unit[d] = std::max(std::min((p[d] - interval_endpoints[d].first -
-                                       ix[d] * delta_x) /
-                                        delta_x,
-                                      1.),
-                             0.);
+        p_unit[d] = std::clamp(
+          std::min((p[d] - interval_endpoints[d].first - ix[d] * delta_x) /
+                     delta_x,
+                   0.,
+                   1.);)
```

Comment:
There is a compile error for the CI. I think you need to use
```suggestion
                   0.,
                   1.));
```
(but I did not test myself).

Reply:
Yes, indeed, this doesn't look right :-)